encryption/ledacrypt/ledacrypt-pkc-cca/impl/avx2.yaml

```diff
@@ -0,0 +1,3 @@
+name: AVX2
+platform: Intel Core i5-6600 3.2  GHz
```

Comment:
`impl/platform` is the programming platform, so here something like `C11`(?).
And `bench/platform` is the actual hardware.

Reply:
Oh right, this has to be one of the early schemes I worked on.